Client/Frontend/Home/Wallpapers/Utilities/WallpaperStorageUtility.swift

```diff
@@ -130,4 +130,28 @@ class WallpaperStorageUtility: WallpaperFilePathProtocol, Loggable {
             browserLog.debug("WallpaperStorageUtility - error deleting folder for \(key)")
         }
     }
+
+    // MARK: - Migration
+    func migrateResources(completion: (Bool) -> Void) {
+        let fileManager = FileManager.default
+        guard let documentPath = fileManager.urls(for: .documentDirectory, in: FileManager.SearchPathDomainMask.userDomainMask).first,
+              let appSupportPath = fileManager.urls(for: .applicationSupportDirectory, in: FileManager.SearchPathDomainMask.userDomainMask).first
+        else {
+            browserLog.debug("Wallpaper migration error")
```

Comment:
What happens for new users, this gets called?

Reply:
Good point. I'll modify this to be better for them. :)

Reply:
Addressed. :)